src/utils/list-of-char.js

```diff
@@ -0,0 +1,25 @@
+const listOfChar = document.getElementById('list-of-char');
+
+const renderCharacters = async () => {
+  fetch('https://rickandmortyapi.com/api/character', {
+    method: 'GET',
+  }).then((resp) => resp.json())
+    .then((json) => json.results)
+    .then((result) => {
+      result.forEach((element) => {
+        const li = document.createElement('li');
+        listOfChar.appendChild(li);
+        li.innerHTML = `
+        <img src=${element.image} alt="${element.name} image" class="mx-auto px-4 pt-4 pb-2 w-75">
```

Comment:
It looks great, and the first page is displaying as expected!
As an Optional change, we could skip the method while using fetch in this function, as GET is the default value already.  
And as a great-to-have kind of change, we could try to implement all the pages, but of course, we can do it at the end if we get some spear time. 
It's getting there, don't you think? :D